### PR TITLE
Beginners guide to add their profile from scratch with step by step guide.

### PR DESCRIPTION
**Pull request by Saha-7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1800"></a>

### CODE REVIEW

**Comment by Saha-7** [comment]
Maintainers can you please check it once !

**Review by 404Nikhil — APPROVED** [review]

**Review by Vyvy-vi — COMMENTED** [review]
The text repeats the steps written a few lines above, and ideally, instructions about adding the profile shouldn't go in the CONTRIBUTING.md file.

**Comment by Saha-7** [comment]
I previously added it to the README.md file but Maintainer @EmmaDawsonDev told me to add it to the CONTRIBUTING.md file.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,6 +135,51 @@ npm run cypress:run
...
+
+- Now turn on Git, choose your own folder/directory to proceed ahead. In my case i choosed Desktop.
+
+- Then type `git clone *The SSH link you copied*`, Press Enter.
+
+- Then get into the project folder through Git CLI. For that type `cd LinkFree` , press Enter.
+
+-Now you are into the LinkFree folder. Now type `git remote -v` ,press enter it will show your origin url.
+
+- Then open VS code by using this command `code `. hit enter.
```
Comment (E-fais) [inline CONTRIBUTING.md:153]: Typo: 
Change this  “code” .  
to 
“ code . “  
To open vs code.

**Review by E-fais — CHANGES_REQUESTED** [review]
Command for opening vs code
“ code  . “

**Comment by Saha-7** [comment]
Thanks, mistakes solved.

**Comment by EmmaDawsonDev** [comment]
Hang tight with this pull request. We (the maintainers) are just reorganising the folder struccture so that this has a proper place to live. Once everything is in place we'll let you know so it can be moved to the correct folder.

**Comment by EmmaDawsonDev** [comment]
Ok, can you move this into the docs/contributing folder and name it contributing-your-profile.md ?

**Comment by Saha-7** [comment]
Sure.

**Comment by Saha-7** [comment]
PR raised in docs/contributing folder #1916

**Pull request closed without merging**